app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,4 @@
 <resources>
     <string name="app_name">MARVEL</string>
     <!-- TODO: Remove or change this placeholder text -->
-    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
Have you checked if this string is used in java code.

Reply:
ya it looks fine.